src/core/templates/show-logo.js

```diff
@@ -1,35 +1,45 @@
 // @ts-check
+import { docLink, showError } from "../../core/utils.js";
 import { html } from "../../core/import-maps.js";
-import { showWarning } from "../../core/utils.js";
 
 const name = "core/templates/show-logo";
 
 /**
+ * Logo mapper. Takes a logo structure and converts it to HTML.
+ *
  * @param {object} logo
  * @param {string} logo.src
  * @param {string} logo.url
  * @param {string} logo.alt
  * @param {string} [logo.id]
  * @param {number} [logo.width]
  * @param {number} [logo.height]
+ * @param {number} index
  */
-export default function showLogo(logo) {
+export default function showLogo(logo, index) {
   /** @type {HTMLAnchorElement} */
-  const a = html`<a href="${logo.url || ""}" class="logo"></a>`;
-  if (!logo.alt) {
-    const msg = "Found spec logo without an `alt` attribute.";
-    showWarning(msg, name, { elements: [a] });
+  const a = html`<a href="${logo.url ?? null}" class="logo"
+    ><img
+      id="${logo.id ?? null}"
+      alt="${logo.alt ?? null}"
+      width="${logo.width ?? null}"
+      height="${logo.height ?? null}"
+      crossorigin
```

Comment:
Why do we want `crossorigin` here?

Reply:
Just for "best current practice". I noticed a lot the injected were cross origin.